### PR TITLE
DOCSP-34812 v1.40.5 Release Notes

### PR DESCRIPTION
**Pull request by kennethdyer**
## DESCRIPTION

Release notes for Compass v1.40.5

## STAGING

[RN](https://preview-mongodbkennethdyer.gatsbyjs.io/compass/DOCSP-34812-compass-1.40.5/release-notes/#compass-1.40.5)

## JIRA

[DOCSP-34812](https://jira.mongodb.org/browse/DOCSP-34812)

## BUILD LOG

* [2023-12-11](https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=65776d967a5cb5fef51451cd)
* [2024-01-18](https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=65a9a2a08aeba96efb394d15)
* [2024-01-19](https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=65aad3b40949a5aff77295e1)

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Is this free of spelling errors?
- [x] Is this free of grammatical errors?
- [x] Is this free of staging / rendering issues?
- [x] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

**Review by jmd-mongo — APPROVED** [review]
Thanks for this addition, @kennethdyer! This one LGTM.

Thanks,
Joe

**Review by Anemy — COMMENTED** [review]
Hi @kennethdyer - There should actually be a few more changes than this, including some bug fixes in this release. 
We're also going to name the release 1.41.0.
This release also isn't out yet, so the date may change depending on if it happens today, or more likely early next week.

I'm not sure exactly which features and fixes we'll want to call out, here's a list though:

Features:
- autoinsert empty document for all fields in the query bar
- add filter to saved connections when there are more than 10 saved [COMPASS-7439](https://jira.mongodb.org/projects/COMPASS/issues/COMPASS-7439)
- implement text search for aggregation stage wizard
- improve stage wizard discoverability and interaction [COMPASS-7350](https://jira.mongodb.org/projects/COMPASS/issues/COMPASS-7350)
- use dark colors for background on initial app loading with dark theme
- remove outdated guide cues [COMPASS-7396](https://jira.mongodb.org/projects/COMPASS/issues/COMPASS-7396)

Fixes:
- include OIDC in $external auth mechanism list [COMPASS-7512](https://jira.mongodb.org/projects/COMPASS/issues/COMPASS-7512)
- properly render syntax errors in embedded shell [COMPASS-7497](https://jira.mongodb.org/projects/COMPASS/issues/COMPASS-7497)
- expanded documents retain state after switching tabs [COMPASS-7318](https://jira.mongodb.org/projects/COMPASS/issues/COMPASS-7318)
- prevent shell container from overlaying sidebar content [COMPASS-7395](https://jira.mongodb.org/projects/COMPASS/issues/COMPASS-7395)
- prevent ai entry button from being submitted when the sort is submitted [COMPASS-7356](https://jira.mongodb.org/projects/COMPASS/issues/COMPASS-7356)

**Comment by kennethdyer** [comment]
@Anemy, Question: Are we abandoning 1.40.5 as a release or is 1.41.0 a new entry for the list?  Should we include COMPASS-7354 in 1.41.0?

**Comment by Anemy** [comment]
@kennethdyer Yes, 1.40.5 is 1.41.0.
 COMPASS-7354 changes up how the tabs look and work. I think it would be fine to include that, yes. Sorry for the late response, missed this during the holidays.

File: source/release-notes.txt [hunk 1]
```diff
@@ -10,6 +10,46 @@ Release Notes
    :depth: 1
    :class: twocols
 
+|compass| 1.41.0
+----------------
+
+*Upcoming*
```
Comment (Anemy) [inline source/release-notes.txt:16]: Just double checking, are we going to update this when we merge? We released 1.41.0 on December 18th

**Review by Anemy — APPROVED** [review]

File: source/release-notes.txt (hunk 1 above)
Comment (kennethdyer) [reply source/release-notes.txt]: Merge pushes the changes into the master branch.  Usually someone on the Server Docs goes through and publishes the updates a couple times a day. 

Fixed

**Pull request merged**